### DISCUSSION TITLE
Different settings for normal and incognito browsing like tracking protection in Firefox and Edge

### DISCUSSION THREAD

**Discussion by Kees1958** [uBlockOrigin/uBlock-issues#2630] [Ideas]
Edge has an option to apply STRICT tracking protection and STRICT security when going inprivate. I thought that Firefox also had an option to enable STRICT tracking protection when incognito.  I really like using balanced mode in normal browsing and strict mode for incognito/inprivate browsing.

I don't know whether uBlockOrigin is aware of its incognito/inprivate mode status, but I do see two versions of uBlockOrigin
![image](https://user-images.githubusercontent.com/30925856/235436707-67759ac9-672f-4a13-bf3f-273f2b336658.png)

Would it be very hard to make uBlockOrigin aware of its inprivate/incognito mode, so it could offer two sets of settings of settings (one for normal browsing and one for inprivate browsing). Having two sets for different browsing goals (remember-me browsing versus incognito browsing), would faciliatate below two use cases:

1. Using more filter lists when browsing incognito 
For visiting bookmarked websites I might want to support their favorite websites by allowing (some) advertisements and have only a few blocklists enabled. For surfing the web incognito mode, it would be nice to add more blocklist.

2. Using dynamic filtering mode when browsing incognito.
On forum's people talk often about 'a balanced and useful' set of filters. More advanced user probably have found their own balanced set and enable dynamic filtering when they switched to incognito mode for more risky browsing. I also like to enable dynamic filtering when going incognito, but I sometimes forget to enable or disable it (which is a nuisance for other people using that PC).

Maybe the members of this community have more use cases (applications) for a set of incognito settings. Please post your thoughts.

**Comment by Kees1958** [comment]
When uBO is aware of being incognito mode, a simple "always use dynamic filtering" option in the settings would achieve use case 2, similar to EDGE (see picture) offers
![image](https://user-images.githubusercontent.com/30925856/235439888-b9eaaeae-f0a3-477e-bb34-5a2d5450f921.png)

**Reply by iam-py-test** [reply]
While that would be a nice feature, IMO it would be difficult to implement. Disabling/enabling dynamic filtering would meaning enabling/disabling Advanced User Mode, which does other things.
Not the uBo dev, just my opinion.

**Comment by iam-py-test** [comment]
Duplicate of https://github.com/uBlockOrigin/uBlock-issues/discussions/2590?

**Reply by Kees1958** [reply]
Nope, #2590 only mentions the first use case, that is why I explicitely added the second use case and also added the simplified version in post 2 (always enable dynamic filtering when incognito)

**Reply by LennyFox** [reply]
I only asked for adding extra blocklists. I now have manualy added allow exceptions for the websites I wanted to sponsor with advertisements, so you can close my suggestion. I will add this comment also in my suggestion,
